webapp/src/actions.js

```diff
@@ -174,6 +175,26 @@ export const list = (reminder = false, listName = 'my') => async (dispatch, getS
     return {data};
 };
 
+export const count = () => async (dispatch, getState) => {
+    let resp;
+    let data;
+    try {
+        resp = await fetch(getPluginServerRoute(getState()) + '/count', Client4.getOptions({
+            method: 'get',
+        }));
+        data = await resp.json();
```

Comment:
```suggestion
    let data;
    try {
        const resp = await fetch(getPluginServerRoute(getState()) + '/count', Client4.getOptions({
            method: 'get',
        }));
        data = await resp.json();
```

Reply:
`resp` is not getting reassigned so I think we can make it a const. Also, it's not being used in the outer scope so we can declare it inside the try block.